src/main/java/fact/io/hdureader/FITS.java

```diff
@@ -83,6 +83,7 @@ public static FITS fromFile(File file) {
     public FITS(URL url) throws IOException {
         this.url = url;
         DataInputStream stream = getUnGzippedDataStream(url);
+        log.info("Reading file: "+url.toString());
```

Comment:
info? or debug?

Reply:
This should be debug and will be in the next commit.